src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Workspace.cs

```diff
@@ -110,9 +68,6 @@ internal void LoadElementExtensions(XmlReader readerOverUnparsedExtensions, int
             _extensions.LoadElementExtensions(readerOverUnparsedExtensions, maxExtensionSize);
         }
 
-        internal void LoadElementExtensions(XmlBuffer buffer)
-        {
-            _extensions.LoadElementExtensions(buffer);
-        }
+        internal void LoadElementExtensions(XmlBuffer buffer) => _extensions.LoadElementExtensions(buffer);
     }
```

Comment:
Convert the rest of the one-line methods to delegate-based method bodies to be consistent.